content-cache/src/nginx_manager.py

```diff
@@ -65,16 +95,40 @@ def initialize() -> None:  # pragma: no cover
     """
     logger.info("Installing and enabling nginx")
     # The install, systemctl enable, and systemctl start are idempotent.
-    return_code, _, stderr = execute_command(["sudo", "apt", "install", "nginx", "-yq"])
+    return_code, _, stderr = execute_command(
+        ["sudo", "apt", "install", "nginx", "lua-resty-core", "-yq"]
```

Comment:
For the `lua-resty-core`, is using this APT package the long term plan?

Reply:
As its part of the distribution, would there be a reason not to use it?